client/balrogclient/api.py

```diff
@@ -118,12 +123,25 @@ def do_request(self, url, data, method):
             logging.debug('Data sent: %s', sanitised_data)
         else:
             logging.debug('Data sent: %s', data)
+        data = json.dumps(data)
         headers = {'Accept-Encoding': 'application/json',
-                   'Accept': 'application/json',
-                   'Content-Type': 'application/json'}
+                    'Accept': 'application/json',
+                    'Content-Type': 'application/json'}
+        if method == ('PUT', 'POST'):
```

Comment:
Needs to be `if method in ('PUT', 'POST'):` otherwise it'll never be True.

Reply:
oops, fixed